src/BalconyBotanica/Core/Algorithm/Recommender.cs

```diff
@@ -47,10 +47,10 @@ public RecommendedPlants Recommend(RecommendedPlants insertedArray, QuizAnswers
                     .OrderBy(x =>
                     {
                         if (x.WateringSchedule == WateringSchedule.MINIMUM)
-                            return 0; // front of array
+                            return 10; // front of array
```

Comment:
We can discuss this tomorrow if you don't quite get how the enum values work

Reply:
yes please!